hooks/src/lib/useCardCVC/useCardCVC.ts

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+import ValidationResult from "../types/ValidationResult";
+import Validation from "../utils/Validation";
+
+interface CVCValidationResult {
+  CVC: string;
+  validationResult: ValidationResult;
+  handleUpdateCVC: (value: string) => void;
+}
+
+export default function useCardCVC(initialValue: string): CVCValidationResult {
+  const [CVC, setCVC] = useState(initialValue);
+  const [validationResult, setValidationResult] = useState<ValidationResult>({
+    isValid: true,
+  });
+
+  const handleUpdateCVC = (value: string) => {
+    setCVC(value);
+
+    if (validateCVC(value)) {
+      setValidationResult({ isValid: true });
+      return;
+    }
+
+    setValidationResult({
+      isValid: false,
+      errorMessage: "CVC 번호는 3자리 숫자로 입력하셔야 합니다.",
+    });
```

Comment:
(minor)
아주 사소한 뉘앙스 차이긴 한데요!
validation 결과(true/false)에 따라 반드시 수행되어야 하는 흐름이 존재하고 각 흐름에서 수행되는 일이 본질적으로 같은 일(setValidationResult)이라면 하나의 문(if else 문)으로 명시 해주는 게 더 읽기에 좋은 것 같아요.
`유효하면 A를 하고 반환해! 아, 그리고 그렇지 않은 경우에는 A'를 해`
이 문장들과 
`유효하면 A를 하고 아니면 A'를 해`
이 문장의 뉘앙스 차이 정도일까요? 마이너한 부분이니 그냥 고려만 해주세요

```suggestion
    if (validateCVC(value)) {
      setValidationResult({ isValid: true });
    } else {
      setValidationResult({
        isValid: false,
        errorMessage: "CVC 번호는 3자리 숫자로 입력하셔야 합니다.",
      });
    }
```

or 

```suggestion
    const result = validateCVC(value)
      ? { isValid: true }
      : { isValid: false, errorMessage: "CVC 번호는 3자리 숫자로 입력하셔야 합니다." }
    
      setValidationResult(result);
```